hat/assets/js/apps/Iaso/domains/instances/compare/hooks/useGetInstanceLogs.ts

```diff
@@ -55,23 +60,107 @@ export const useGetInstanceLogs = (
 };
 
 export const useGetInstanceLogDetail = (
-    logId: string | undefined,
-): UseQueryResult<Record<string, any> | undefined, Error> => {
-    const queryKey: any[] = ['instanceLogDetail', logId];
-    return useSnackQuery({
-        queryKey,
-        queryFn: () => getInstanceLogDetail(logId),
-        options: {
-            enabled: Boolean(logId),
-            select: data => {
-                if (data) {
-                    return data.new_value[0].fields;
-                }
+    logA: string | undefined,
+    logB: string | undefined,
+): UseQueryResult<InstanceLogFileContent, Error> => {
+    const [
+        { data: instanceLogADetail, isFetching: isInstanceLogAFetching },
+        { data: instanceLogBDetail, isFetching: isInstanceLogBFetching },
+    ] = useSnackQueries([
+        {
+            queryKey: ['instanceLogADetail', logA],
+            queryFn: () => getInstanceLogDetail(logA),
+            snackErrorMsg: MESSAGES.fetchLogDetailError,
+            dispatchOnError: false,
+            options: {
+                enabled: Boolean(logA),
+                select: data => {
+                    if (data) {
+                        return data.new_value[0].fields;
```

Comment:
`data` has no type

Reply:
nitpick I find it clearer to end the `select`with what you actually want to return:

```
select: data => {
    if(!data) return data // if data evaluates to false, it's undefined
    return data.new_value[0].fields;
}
```